activities-bot/src/utils.mjs

```diff
@@ -252,3 +252,17 @@ export async function postEODMessage({ github, slack, updates }) {
     getHumanReadableUpdates(updates, slack, github),
   );
 }
+
+export async function withRetry(method, { attempts }) {
+  while (attempts) {
+    try {
+      return await method();
+    } catch (error) {
+      attempts -= 1;
```

Comment:
For the retries to work, do add a throw statement inside fetch when !ok, or lets create a wrapper around fetch which does that

Reply:
Done. Original issue was for non HTTP errors. This should now cover for HTTP errors too.